content/docs/SUSHI/migration/_index.md

```diff
@@ -92,6 +92,11 @@ Move the **./input/fsh/config.yaml** file to the top-level folder, and rename it
 
 You can now run SUSHI 1.0. You may get further instructions in the form of error messages.
 
+{{% alert title="Note" color="primary" %}}
+Although SUSHI 2.0 is available for use, if you are migrating from a pre-1.0.0 version of SUSHI, we recommend first updating to the latest SUSHI 1.0 version of SUSHI (this is [SUSHI 1.3.2](https://github.com/FHIR/sushi/releases/tag/v1.3.2)). After resolving all errors and warnings in that version, only then should you migrate to a SUSHI 2.0 release.
+{{% /alert %}}
```

Comment:
Should you note the command to install an older version since some might not know it?  E.g. `npm install -g fsh-sushi@1.3.2`

Reply:
Noted in https://github.com/FSHSchool/site/pull/42/commits/2ee12fa49e1511aae4752703f90b96ad259f984f